.travis.yml

```diff
@@ -1,33 +1,66 @@
+sudo: required
+
 language:
   - cpp
-#  - objective-c
 
-dist: trusty
```

Comment:
Does removing this change anything?

Reply:
no, it still uses trusty after

Reply:
trusty being supported until april 2019 (six months ahead) I guess travis will update to something else (xenial, 16.04, up to april 2021) around this date